land_use/admin.py

```diff
@@ -0,0 +1,67 @@
+from django.contrib import admin
+from land_use import models
+
+
+class LandUseClassesAdmin(admin.ModelAdmin):
+    """Django administrator `models.LandUseClasses` data."""
+
+    list_display = (
+        'id',
+        'sg_uf',
+        'no_ti',
+        'co_funai',
+        'dt_homologada',
+        'ds_cr',
+        'co_cr',
+        'nu_ano',
+        'no_estagio',
+        'no_satelites',
+        'nu_resolucoes',
+        'dt_imagens',
+        'nu_area_km2',
+        'nu_area_ha',
+        'dt_cadastro',
+        # 'geom',
+    )
+
+    fields = list_display
+
+    search_fields = list_display
```

Comment:
Search field for every field? 
It might be less fields.

Reply:
Adjusted search_fields.